edx_name_affirmation/api.py

```diff
@@ -212,3 +214,38 @@ def update_is_verified_status(
         )
     )
     log.info(log_msg)
+
+
+def create_verified_name_config(user, use_verified_name_for_certs=None):
+    """
+    Create verified name configuration for the given user.
+    Arguments:
+        * `user` (User object)
+        * `use_verified_name_for_certs` (bool): If True, certificates will prioritize
+          the user's verified name over their profile name.
+    """
+    fields = {'user': user}
+    if use_verified_name_for_certs is not None:
+        fields['use_verified_name_for_certs'] = use_verified_name_for_certs
+
+    config_obj = VerifiedNameConfig.objects.create(**fields)
```

Comment:
The config model has an `enabled` field. Is this something that we want to use? For example, I could imagine that we would want to mark a new config as `enabled` upon creation, but that also means we would want to mark any older configs as not enabled.

Reply:
For now, I don't believe this flag needs to be used. If you think we should always default to `enabled`, my opinion would be fine with it.